### ISSUE TITLE
Simulated login flow is absent

### ISSUE THREAD

**Issue by Julian-O** [Julian-O/Instagram-API-python#5]
In the PHP library is a dire warning marked "SUPER IMPORTANT" [on line 589](https://github.com/mgp25/Instagram-API/blob/098266cebb7427ddc5e088fff0c7f45e541e34b4/src/Instagram.php) that describes how, after logging in, you must make a sequence of 14 (current count) unnecessary calls to Instagram, to properly simulate how a phone refreshes itself.

It says if you don't do this Instagram 
> WILL detect you as a bot and silently BLOCK features or even ban you.

The LevPasha version of this library, that this was forked from, made some of those calls - I think it was an out of date list, because several resulted in errors. (The errors were simply ignored, but the new exception regime meant in this fork, the errors quite rightly prevented logging in.)

The calls were simply removed, logging in became considerably faster, and to date, Instagram haven't stomped on my account, but I am living life on the edge.

Consider returning the simulated calls to the log in flow.

**Comment by roperi** [comment]
Hi. I think it is better to have the simulated calls than not having them. The new exception regime should be reworked. I am kind of noob so ..is your new exception system hard to change?

**Comment by Julian-O** [comment]
Yeah, they should be added back - or at the very least become an option. This is mainly a note to myself, and to let other users know that (unless they want to do the calls manually) it might be an issue.

The API now uses exceptions in the standard pythonic way. In the past, it relied on return codes that are painful to check and often forgotten. Even the example code didn't check results - which means you could get in trouble when, for example, you thought you were logged in, but the password had failed.

If the simulated calls get added back in, they should either filter out the calls that don't make sense, or wrap them in exception handlers to explicitly indicate that we don't care if they fail.
